hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/TestHoodieSparkSqlWriter.scala

```diff
@@ -1232,6 +1232,7 @@ class TestHoodieSparkSqlWriter {
     val beforeWriteInstant = Instant.now()
 
     // set local timezone to America/Los_Angeles(UTC-7)
+    val defaultTimezone = TimeZone.getDefault
     TimeZone.setDefault(TimeZone.getTimeZone("America/Los_Angeles"))
```

Comment:
Okay, my fault, we should never allows setting timezone explicitly in the code, the whole JVM process could be affected.

Reply:
No worries.  This should fix the test issue.